refactor(editor): Dark theme improvements (no-changelog)

netroy opened the pull request.

netroy committed:
refactor(editor): Remove `getStyleTokenValue` (no-changelog)

netroy committed:
use css variable in code editor as well

mutdmour reviewed (dismissed):
Looks good. tested everything, LGTM. Found unrelated bug.. will fix seperately

mutdmour committed:
fix: Update colors of syntax errors (#7575)

Github issue / Community forum post (link here to close automatically):
<img width="201" alt="Screenshot 2023-11-01 at 16 58 21"
src="https://github.com/n8n-io/n8n/assets/4711238/15fcbe25-2dec-4b1d-8d27-f5dc186bf2ac">
<img width="125" alt="Screenshot 2023-11-01 at 16 59 37"
src="https://github.com/n8n-io/n8n/assets/4711238/414d9122-79bd-4bd6-b188-b8f26091f1ce">

janober commented:
Got released with [`n8n@1.15.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.15.1)
